versioned_docs/version-3.13/whats-new.md

```diff
@@ -0,0 +1,81 @@
+---
+title: What's New in RabbitMQ 3.13
+---
+<!--
+Copyright (c) 2024 Broadcom. All Rights Reserved. The term "Broadcom" refers
+to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+import {
+  RabbitMQServerVersion,
+} from '@site/src/components/RabbitMQServer';
+
+import MQTTLogo from '@site/static/img/mqtt-logo/mqtt-ver.svg';
+import KhepriLogo from '@site/static/img/khepri-logo.svg';
+
+# What's New in RabbitMQ 3.13
+
+RabbitMQ 3.13 is a new release branch [published on 22 Feb
+2024](/blog/2024/03/11/rabbitmq-3.13.0-announcement). <strong>RabbitMQ
+<RabbitMQServerVersion/></strong> is the latest patch release for this release
+branch. See the [Release Information section](/release-information) to learn
+more about all patch releases for this release branch.
+
+## MQTT 5.0 support
+
+<MQTTLogo className="floating-logo"/>
+
+MQTT is the standard protocol for the Internet of Things (IoT).
+
+IoT remote devices can have a poor network quality when connecting to the
+broker. Therefore, MQTT is lightweight: MQTT protocol headers are small to save
+network bandwidth. Since IoT devices might often disconnect and reconnect -
+imagine a car driving through a tunnel - MQTT is also efficient: Clients are
+connected and authenticated with a handshake that is shorter compared to other
+messaging protocols.
+
+[MQTT 5.0](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html) is
+the latest version of the protocol and [is now supported by
+RabbitMQ 3.13.0](/blog/2023/07/21/mqtt5)!
+
+## Stream filtering
+
+[Stream filtering](/blog/2023/10/16/stream-filtering) allows to save bandwidth
+between the broker and consuming applications when those applications need only
+a subset of the messages of a stream.
+
```

Comment:
I would get rid of the word "allows to" and just say "With the new Stream filtering feature, you can save bandwidth......"